### DISCUSSION TITLE
extend Evoke Gerund

### DISCUSSION THREAD

**Discussion by tangentstorm** [jsoftware/jsource#248] [Ideas]
I'd like to propose the extension for the Evoke Gerund (`` `: ``) conjunction, described on page [Extended forks](https://code.jsoftware.com/wiki/User:Igor_Zhuravlov/Extended_forks). Following is example of such extension usage, e.g. for 3-fork ("9" is selected randomly for example only):

```
   NB. proposed evoke by extended Evoke Gerund (`:) functionality
   2 (+`%:`-`%`^`*`:9) 4
0.00379719j0.00219231

   NB. evoke using the model adverb from script supplied with page 'Extended forks'
   2 (+`%:`-`%`^`* fork3) 4
0.00379719j0.00219231
```

In gerund, Cap (`[:`) (at leftmost position in execution tree) and Self-Reference (`$:`) verbs support is desirable.

-- Igor Zhuravlov <2009-12-26T19:30:00Z>

**Comment by jip** [comment]
Cross-reference: [[JWiki] System/Interpreter/Requests](https://code.jsoftware.com/wiki/System/Interpreter/Requests#extend_Evoke_Gerund), Igor Zhuravlov, Sat Dec 26 19:30:00 UTC 2009.

**Comment by HenryHRich** [comment]
The idea is radical and very general.  IMO no one but the author would use it.
